lib/mayaUsd/resources/ae/usdschemabase/ae_template.py

```diff
@@ -754,26 +871,27 @@ def sectionNameFromSchema(self, schemaTypeName):
         name is a pretty name used in the UI.'''
 
         # List of special rules for adjusting the base schema names.
-        prefixToAdjust = {
-            'UsdAbc' : '',
-            'UsdGeomGprim' : 'GeometricPrim',
-            'UsdGeom' : '',
-            'UsdHydra' : '',
-            'UsdImagingGL' : '',
-            'UsdLux' : '',
-            'UsdMedia' : '',
-            'UsdRender' : '',
-            'UsdRi' : '',
-            'UsdShade' : '',
-            'UsdSkelAnimation' : 'SkelAnimation',
-            'UsdSkelBlendShape': 'BlendShape',
-            'UsdSkelSkeleton': 'Skeleton',
-            'UsdSkelRoot' : 'SkelRoot', 
-            'UsdUI' : '',
-            'UsdUtils' : '',
-            'UsdVol' : ''
-        }
-        for p, r in prefixToAdjust.items():
+        prefixToAdjust = [
+            ('UsdAbc', ''),
+            ('UsdGeomGprim', 'GeometricPrim'),
+            ('UsdGeomImageable', mel.eval('uiRes(\"m_AEdagNodeTemplate.kDisplay\");')),
+            ('UsdGeom', ''),
+            ('UsdHydra', ''),
+            ('UsdImagingGL', ''),
+            ('UsdLux', ''),
+            ('UsdMedia', ''),
+            ('UsdRender', ''),
+            ('UsdRi', ''),
+            ('UsdShade', ''),
+            ('UsdSkelAnimation', 'SkelAnimation'),
+            ('UsdSkelBlendShape', 'BlendShape'),
+            ('UsdSkelSkeleton', 'Skeleton'),
+            ('UsdSkelRoot', 'SkelRoot'), 
+            ('UsdUI', ''),
+            ('UsdUtils', ''),
+            ('UsdVol', '')
+        ]
+        for p, r in prefixToAdjust:
```

Comment:
Needed to change this to a list of tuples since in python 2.7 the dictionaries were not ordered. So sometimes the `UsdGeom` would come before the `UsdGeomImageable` and then it wouldn't change `Imageable` to `Display`.